tests/e2e/config/setup.js

```diff
@@ -39,14 +39,19 @@ module.exports = async ( globalConfig ) => {
 			createCoupons(),
 			createCategories(),
 			createShippingZones(),
+			createProductAttributes(),
 			enablePaymentGateways(),
-			setupPageSettings(),
```

Comment:
@opr is there a reason this was removed?

Reply:
I can't remember, can you add it back and does it still work fine if you do?